client/users/user-profile.tsx

```diff
@@ -596,7 +622,8 @@ function TotalGamesEntry({ race, wins, losses }: { race: RaceChar; wins: number;
       <div>
         <TotalGamesText>{wins + losses}</TotalGamesText>
         <WinLossText>
-          {wins} W &ndash; {losses} L &ndash; {winrate}%
+          {wins} {t('game.results.winAcronym', 'W')} &ndash; {losses}{' '}
```

Comment:
not sure a single letter qualifies as an acronym :) Maybe `winShort` and `lossShort`